<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>341</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>RGB Value</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="sliderR">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelR">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderG">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelG">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderB">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="labelB">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>HSV Value</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="sliderH">
       <property name="maximum">
        <number>359</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelH">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderS">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelS">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderV">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="labelV">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelR</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelG</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelB</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderH</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelH</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderS</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelS</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderV</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelV</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
